### PR TITLE
Add 'current' option for `golemcli tasks show` to display only current (ongoing) tasks

### LINKED ISSUE #3106: Add `current` additional argument in `tasks` cli call
Print current task id, task state (progress; is it computing or being computed), task cost, task settings (resolution; frames; subtask; timeouts)

### PR DESCRIPTION
**Pull request by dybi**
resolves: https://github.com/golemfactory/golem/issues/3106

### CODE REVIEW

**Comment by dybi** [comment]
@jamuszyn - could you please check what format is displayed in GUI for status, ie.:
"notStarted" or "Not started", "waiting" or "Waiting", etc.

EDIT:
checked offline - everything's crystal clear now.

File: golem/interface/client/tasks.py [hunk 1]
```diff
@@ -26,6 +27,13 @@ class Tasks:
     unsupport_reasons_table_headers = ['reason', 'no of tasks',
                                        'avg for all tasks']
 
+    current_task_states = [
```
Comment (etam) [inline golem/interface/client/tasks.py]: `TaskStatus` has `is_completed` method. Maybe you could add a new, similar one, and use it instead?

Comment (dybi) [reply golem/interface/client/tasks.py]: I was wondering if it would be ok to do this. Sure :+1:

Comment (dybi) [reply golem/interface/client/tasks.py]: done

File: golem/interface/client/tasks.py [hunk 2]
```diff
@@ -57,20 +58,34 @@ class Tasks:
...
+    def show(self, id, sort, current):
 
         deferred = Tasks.client.get_tasks(id)
         result = sync_wait(deferred)
 
         if not id:
             values = []
 
+            if current:
+                result = [t for t in result
```
Comment (Wiezzel) [inline golem/interface/client/tasks.py:86]: `result or []` in line 89 suggests that `result` might be `None`. It should be handled here as well.

**Review by etam — APPROVED** [review]
ok

File: golem/interface/client/tasks.py (hunk 2 above)
Comment (etam) [reply golem/interface/client/tasks.py:86]: Or line 80 can be changed to `result = sync_wait(deferred) or []`

**Review by etam — CHANGES_REQUESTED** [review]
@Wiezzel had a good point

File: golem/interface/client/tasks.py (hunk 2 above)
Comment (dybi) [reply golem/interface/client/tasks.py:86]: quite frankly, it shouldn't be; if you thoroughly analyze `def get_tasks(self, task_id: Optional[str] = None)` from `golem.client`, you will see that `None` can be returned only when `task_id` has `True` value in boolean context. But when this is the case, whole `if` from line 82 is skipped. 

So it seems that `or []` from line 89 should be removed.
If you insist, I can add handling of `None`, but so far I feel more inclined towards removing `or []`. @Wiezzel, what would be your preference here?

Anyway, thanks for pointing this out :)

Comment (Wiezzel) [reply golem/interface/client/tasks.py:86]: If you're sure this won't happen, just remove the `or []`. I'm fine with this.

Comment (dybi) [reply golem/interface/client/tasks.py:86]: @Wiezzel @etam removed; also I have added a unit test to make sure we will notice if logic connected with `None` ever changes

**Review by etam — APPROVED** [review]
